packages/insomnia-app/app/ui/css/components/graph-ql-explorer.less

```
@@ -12,6 +12,9 @@
   background: var(--color-bg);
   box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.4);
   color: var(--color-font);
+  -webkit-user-select: text;
+  -moz-user-select: text;
+  user-select: text;
```

Because this is an electron only app and we don't need to think about other browsers yet, so it should be safe to omit the `-webkit` and `-moz` styles, just keeping the regular `user-select`. 🙂 

```suggestion
  user-select: text;
```

---

Thank you for the suggestion @develohpanda ! I just pushed the change.